internal/engine/compiler/engine.go

```diff
@@ -966,13 +966,13 @@ func newEngine(enabledFeatures api.CoreFeatures, fileCache filecache.Cache) *eng
 //
 // By declaring these values as `var`, slices created via `make([]..., var)`
 // will never be allocated on stack [1]. This means accessing these slices via
-// raw pointers is safe: As of version 1.18, Go's garbage collector never relocates
+// raw pointers is safe: As of version 1.21, Go's garbage collector never relocates
```

Comment:
unless I am misunderstanding, this may stay at 1.18, because the fact we are updating does not change that Go's GC has that behavior since 1.18

Reply:
No, not necessary to change, but I think it adds information: it's 1.21 and Go is still a non moving GC.

Reply:
I think I misread "as of" as "since" :)

Reply:
I can picture us on the Go 1.42 release "the year is 2044, Go still doesn't have a moving GC"